test/parse-result.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Tests for parse result namespace elements, including all their
+ * convenience properties and methods.
+ */
+
+import chai, {Assertion, expect} from 'chai';
+
+import minim from 'minim';
+import minimApiDescription from 'minim-api-description';
+import minimParseResult from '../src/parse-result';
+
+const namespace = minim.namespace()
+  .use(minimApiDescription)
+  .use(minimParseResult);
+
+const Category = namespace.getElementClass('category');
+const ParseResult = namespace.getElementClass('parseResult');
+const Annotation = namespace.getElementClass('annotation');
+const SourceMap = namespace.getElementClass('sourceMap');
```

Comment:
Out of context, but just thinking out loud. I seem to be doing this a lot myself and I think we should think of an easy way to do it, especially given ES6 does destructuring. 

```js
{ Category, ParseResult, Annotation, SourceMap } = namespace.elements();
```